### PR TITLE
11333 - new Security page

### LINKED ISSUE #11333: Website Redesign - Security
As a User,
I want to visit the ReportStream Security Page,
So that I can see understand how ReportStream is keeping my data safe

### Designs
- Figma Link https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=3097-7162&mode=dev

### Verification Steps
- Code Review
- Trial / Dev environment signoff by Product / Design
- Staging environment signoff by Product / Design / Stakeholders 
- Production environment signoff by Engineering, Product and Design

### Acceptance Criteria
- [ ] Storybook update required
- [ ] Unit Tests
- [x] Manual Testing Steps
- [x] Update link to be /about/security

Dependencies:
- Use Feature Branch `webx-sprint-77-feature-branch` as the base branch
- Designs - https://www.figma.com/file/Du7feM8huRDAAZCQcUIj93/Website-Redesign?type=design&node-id=3097-7162&mode=dev
- Navigation links

Additional Details:
- The deployment strategy will follow the general deployment strategy for the Website Redesign EPIC. We will deliver this page in isolation from the other pages.

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

Is the new Security page

Test Steps:
1. Log into RS
2. Navigate to About -> Security

## Changes
Screenshot 2023-09-26 at 10 18 06 AM

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #11333

### CODE REVIEW

File: frontend-react/src/AppRouter.tsx [hunk 1]
```diff
@@ -249,6 +256,14 @@ export const appRoutes: RouteObject[] = [
                     },
                 ],
             },
+            {
+                path: "/product/release-notes",
```
Comment (jpandersen87) [inline frontend-react/src/AppRouter.tsx:260]: Is release-notes staying under the product sub-directory or does it need to move to /about?

Comment (penny-lischer) [reply frontend-react/src/AppRouter.tsx:260]: @etanb is working on release notes and will fix in another PR

**Review by jpandersen87 — APPROVED** [review]

**Comment by jillian-hammer** [comment]
hi @penny-lischer thanks! are you able to push to a trial enviro for audrey and i to review?

**Review by jillian-hammer — APPROVED** [review]
LGTM thanks @penny-lischer

**Review by etanb — APPROVED** [review]

**Review by audreykwr — APPROVED** [review]

**Pull request merged**